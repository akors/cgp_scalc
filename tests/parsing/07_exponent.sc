2^2
2.0^3.0

1.412135^2.0 # sqrt of two to the power of two
16^(1/2)# sqrt of 16

2^2^3
2^(2^3)
(2^2)^3

