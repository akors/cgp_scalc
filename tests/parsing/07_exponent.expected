4
8
1.99413
4
256
256
64
